spec/features/navigation_spec.rb

```diff
@@ -84,20 +121,22 @@
       visit '/merchant'
       expect(page).to have_content("The page you were looking for doesn't exist.")
 
-      within 'nav' do 
-        click_link "Login"
-      end
-
-      expect(current_path).to eq('/login')
-
-      within 'nav' do
-        click_link "Register"
-      end
-
-      expect(current_path).to eq('/register')
-
       visit '/cart'
       expect(page).to have_content("The page you were looking for doesn't exist.")
     end
+    it "I can see the link to my profile page and a link to logout" do
+      user = create(:admin_user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+      visit "/"
+
+      expect(page).to have_link("Profile")
+      expect(page).to have_link("Logout")
+      expect(page).to have_no_content("Login") 
+      expect(page).to have_no_content("Register")
+      expect(page).to have_link("Dashboard") 
```

Comment:
Same as last comment, except `expect(page).to have_link("Dashboard", href: "/admin")`